src/khoj/interface/email/magic_link.html

```diff
@@ -1,17 +1,43 @@
 <!DOCTYPE html>
-<html>
-    <head>
-        <title>Welcome to Khoj</title>
-    </head>
-<body>
-<body style="font-family: 'Verdana', sans-serif; font-weight: 400; font-style: normal; padding: 0; text-align: left; width: 600px; margin: 20px auto;">
-<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-<a class="logo" href="https://khoj.dev" target="_blank" style="text-decoration: none; text-decoration: underline dotted;">
-    <img src="https://assets.khoj.dev/khoj_logo.png" alt="Khoj Logo" style="width: 100px;">
-</a>
-<p style="color: #333; font-size: medium; margin-top: 20px; padding: 0; line-height: 1.5;">Hi! <a href="{{ link }}" target="_blank" style="text-decoration: none; text-decoration: underline dotted;">Click here to sign in on this browser.</a></p>
-
-<p style="color: #333; font-size: large; margin-top: 20px; padding: 0; line-height: 1.5;">- The Khoj Team</p>
+<html lang="en">
 
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Khoj</title>
+</head>
+
+<body
+    style="font-family: 'Arial', sans-serif; line-height: 1.6; color: #333; max-width: 600px; margin: 0 auto; padding: 20px; background-color: #f5f5f5;">
+    <div
+        style="background-color: #ffffff; border-radius: 10px; box-shadow: 0 0 20px rgba(0, 0, 0, 0.1); padding: 30px;">
+        <a href="https://khoj.dev" target="_blank"
+            style="display: block; text-align: center; margin-bottom: 20px; text-decoration: none;">
+            <img src="https://assets.khoj.dev/khoj_logo.png" alt="Khoj Logo" style="width: 120px;">
+        </a>
+
+        <p style="font-size: 16px; color: #333; margin-bottom: 20px;">Use this code to login to Khoj:</p>
+
+        <h1 style="font-size: 24px; color: #2c3e50; margin-bottom: 20px; text-align: center;">{{ code }}</h1>
+
+        <p style="font-size: 16px; color: #333; margin-bottom: 20px;">It will be valid for 5 minutes.</p>
+
+        <p style="font-size: 16px; color: #333; margin-bottom: 20px;">Alternatively, <a href="{{ link }}" target="_blank"
+                style="color: #FFA07A; text-decoration: none; font-weight: bold;">Click here to sign in on this
+                browser.</a></p>
+
+        <p style="font-size: 16px; color: #333; margin-bottom: 20px;">You're about to get a whole lot more productive.
+        </p>
+
+        <div style="font-size: 18px; font-weight: bold; margin-top: 30px; text-align: right;">- The Khoj Team</div>
```

Comment:
Should we keep the font smaller and not bold for the sign-off text?

```suggestion
        <div style="font-size: 14px; color: #555; margin-top: 30px;">- The Khoj Team</div>
```